proxy/src/proxy.rs

```diff
@@ -38,6 +38,15 @@ static NUM_BYTES_PROXIED_COUNTER: Lazy<IntCounter> = Lazy::new(|| {
     .unwrap()
 });
 
+static NUM_BYTES_PROXIED_PER_CLIENT_COUNTER: Lazy<IntCounterVec> = Lazy::new(|| {
```

Comment:
@funbringer , what do you think of having two different numbers for income and outcome traffic? Why am I asking? AWS income traffic costs 0 and these numbers can help us in the future to understand how can we make our solutions cheaper or more expensive based on these numbers.

https://aws.amazon.com/en/ec2/pricing/on-demand/

Reply:
Good idea!